DDS-1680: AddFailedFiles endpoint to insert files if failed (#1472)
ScilifelabDataCentre/dds_web: SPRINTLOG.md

- Use full DDS name in MOTD email subject ([#1477](https://github.com/ScilifelabDataCentre/dds_web/pull/1477))
- Add flag --verify-checksum to the comand in email template ([#1478])(https://github.com/ScilifelabDataCentre/dds_web/pull/1478)
- Improved email layout; Highlighted information and commands when project is released ([#1479])(https://github.com/ScilifelabDataCentre/dds_web/pull/1479)

> - Add endpoint for updating failed files in database ([#1472])(https://github.com/ScilifelabDataCentre/dds_web/pull/1472)

Reviewer: ```suggestion
- New endpoint `UpdateFailedFiles` for adding failed files to database ([#1472])(https://github.com/ScilifelabDataCentre/dds_web/pull/1472)
```
